vispy/color/color_array.py

```diff
@@ -124,9 +128,24 @@ class ColorArray(object):
     Under the hood, this class stores data in RGBA format suitable for use
     on the GPU.
     """
-    def __init__(self, color='black', alpha=None, clip=False):
+    def __init__(self, color=None, alpha=None,
+                 clip=False, color_space='rgb'):
+
+        # if color is RGB, then set the default color to black
+        if color_space == 'rgb':
+            # by default, if the color is None, we pick black
+            color = 'black' if color is None else color
+        elif color_space == 'hsv':
+            color = (0, 0, 0) if color is None else color
+            # if the color space is hsv, convert hsv to rgb
+            color = _hsv_to_rgb(color)
+        else:
+            raise ValueError('color_space should be either "rgb" or'
+                             '"hsv", it is ' + color_space)
+
         """Parse input type, and set attribute"""
```

Comment:
This line is odd. As a doc string, you might think it should stay at the top of the function, but there's already a doc string before the function. So maybe it should be a comment? (This is not a question about this PR specifically, BTW.)